rugeGerritsen/sdk-nrf
# Add BLE controller flash driver

To test the driver

1. go to `fw-nrfconnect-zephyr/samples/drivers/soc_flash_nrf`
2. configure the project as usual, select the nrfxlib BLE controller, logging, ...
3. ensure that "Device drivers" => "Flash hardware support" => "Nordic semiconductor flash driver" has selected "Nordic BLE controller flash driver"
compile and run the example.
